src/meson.build

```diff
@@ -0,0 +1,126 @@
+describe_h = configure_file(
+  output: 'describe.h',
+  configuration: configuration_data({
+    'EPICS_HOST_ARCH': f'"@epics_host_arch@"',
+    'T_A': f'"@epics_target_arch@"',
+    'OS_CLASS': f'"@epics_os_class@"',
+    'CMPLR_CLASS': f'"@epics_cmplr_class@"',
+  }),
+)
+
+vcs_h = configure_file(
+  output: 'pvxsVCS.h',
+  configuration: configuration_data({
+  }),
+)
+
+version = meson.project_version().split('.')
+version_num_h = configure_file(
+  output: 'versionNum.h',
+  configuration: configuration_data({
+    'PVXS_MAJOR_VERSION': version[0],
+    'PVXS_MINOR_VERSION': version[1],
+    'PVXS_MAINTENANCE_VERSION': version[2],
+  }),
+)
+
+# HACK: this installs generated headers in a 'pvxs' subdir
+# See: https://github.com/mesonbuild/meson/issues/2320
+fs = import('fs')
+run_command(
+  [
+    'sh',
+    '-c',
+    'mkdir @0@/pvxs; cp @1@/@2@ @1@/@3@ @1@/@4@ @0@/pvxs/'.format(
+      meson.current_build_dir(),
+      fs.parent(meson.current_build_dir()),
+      version_num_h,
+      vcs_h,
+      describe_h,
```

Comment:
As far as I can tell, the EPICS build does not install `pvxsVCS.h` or `describe.h`.

Reply:
You're right! I'll remove them from the `install_headers` below